### DISCUSSION TITLE
Should blanket be for Windows?

### DISCUSSION THREAD

**Discussion by CsarChvz** [rafaelmardojai/blanket#133] [Ideas]
I think a blanket adaptation in Windows could be a good advance

**Comment by rafaelmardojai** [comment]
My target is Linux/GNOME, but as I already said in #117, if someone wants to package it for Windows it's welcome.

**Comment by terenc3** [comment]
@CsarChvz I've create a web version of Blanket, which might suits your needs https://apps.roanapur.de/blanket/

**Reply by archisman-panigrahi** [reply]
It may be possible to package it as a standalone app using https://nwjs.io/

**Reply by terenc3** [reply]
Yes it totally is possible, maybe i will invest some time in it
